qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -66,6 +66,7 @@
    XGate
    YGate
    ZGate
+   BRGGate
```

Comment:
I believe this will be auto-imported correctly into `qiskit.circuit.library`, but will  need to be added to the module docstring in `qiskit/circuit/library/__init__.py` to show up in the docs.